`mmakebuildtree --newdriver` fails with `make -j`

`mmakebuildtree --newdriver` fails when using `-j` make option:
```
$ mmakebuildtree --newdriver
$ make -C x86_64_linux-5.19.0-23-generic/ -j5
make: Entering directory '/home/sasha/work/level5/onload/build/x86_64_linux-5.19.0-23-generic'
make -C ../.. KPATH=/usr/src/linux-headers-5.19.0-23-generic KBUILDTOP=/home/sasha/work/level5/onload/build/x86_64_linux-5.19.0-23-generic kernel
make[1]: Entering directory '/home/sasha/work/level5/onload'
make[2]: Entering directory '/usr/src/linux-headers-5.19.0-23-generic'
warning: the compiler differs from the one used to build the kernel
  The kernel was built by: x86_64-linux-gnu-gcc-12 (Ubuntu 12.2.0-3ubuntu1) 12.2.0
  You are using:           gcc (Ubuntu 12.2.0-3ubuntu1) 12.2.0
(/home/sasha/work/level5/onload/src/driver/linux_resource/kernel_compat.sh -k /usr/src/linux-headers-5.19.0-23-generic -q > /home/sasha/work/level5/onload/build/x86_64_linux-5.19.0-23-generic/src/driver/linux_resource/autocompat.h) || (rm -f /home/sasha/work/level5/onload/build/x86_64_linux-5.19.0-23-generic/src/driver/linux_resource/autocompat.h && false)
/home/sasha/work/level5/onload/src/driver/linux_net/drivers/net/ethernet/sfc/Makefile:70: SFE4001/Falcon is no longer supported
  GENERATE /home/sasha/work/level5/onload/build/x86_64_linux-5.19.0-23-generic/src/lib/ciul/onload_version.h
T=$(mktemp -p /home/sasha/work/level5/onload/build/x86_64_linux-5.19.0-23-generic/src/lib/ciul/.); \
(cd /home/sasha/work/level5/onload/src; /home/sasha/work/level5/onload/src/../scripts/onload_version_gen) > $T; \
if diff -q /home/sasha/work/level5/onload/build/x86_64_linux-5.19.0-23-generic/src/lib/ciul/onload_version.h $T 2>/dev/null; \
        then echo "Not updating onload_version.h"; \
        else echo "Updated onload_version.h"; \
             mv $T /home/sasha/work/level5/onload/build/x86_64_linux-5.19.0-23-generic/src/lib/ciul/onload_version.h; \
fi; \
rm -f $T
  CC      /home/sasha/work/level5/onload/build/x86_64_linux-5.19.0-23-generic/src/lib/ciul/checksum.o
  CC      /home/sasha/work/level5/onload/build/x86_64_linux-5.19.0-23-generic/src/lib/ciul/ef100_event.o
Updated onload_version.h
  CC      /home/sasha/work/level5/onload/build/x86_64_linux-5.19.0-23-generic/src/lib/ciul/ef100_vi.o
  UPD     /home/sasha/work/level5/onload/build/x86_64_linux-5.19.0-23-generic/src/driver/linux_net/drivers/net/ethernet/sfc/config.h
In file included from /home/sasha/work/level5/onload/src/include/ci/tools/sysdep.h:55,
                 from /home/sasha/work/level5/onload/src/include/ci/tools/log.h:39,
                 from /home/sasha/work/level5/onload/src/include/ci/tools.h:20,
                 from /home/sasha/work/level5/onload/src/lib/ciul/memcpy_to_io.h:18,
                 from /home/sasha/work/level5/onload/src/lib/ciul/ef100_vi.c:8:
/home/sasha/work/level5/onload/src/include/ci/tools/platform/linux_kernel.h:75:11: fatal error: stdarg.h: No such file or directory
   75 | # include <stdarg.h>
      |           ^~~~~~~~~~
compilation terminated.
```
It all builds without `-j`.  It looks like `EFRM_HAS_LINUX_STDARG_H` is not set properly (yet?).  I guess there may be issues with other compat macros.

@rhughes-xilinx

--- a/Makefile
+++ b/Makefile
@@ -83,6 +83,7 @@ mkdirs:
 # Define the high-level dependencies between libraries:
 $(obj)/src/driver/linux_resource: $(AUTOCOMPAT) mkdirs
 $(obj)/src/lib/transport/ip: $(AUTOCOMPAT)
+$(obj)/src/lib/ciul: $(AUTOCOMPAT)
 $(obj)/src/lib/citools: $(AUTOCOMPAT)
 $(obj)/src/lib/cplane: $(AUTOCOMPAT) $(obj)/src/lib/ciul
 $(obj)/src/driver/linux_char: $(AUTOCOMPAT)

